diff --git a/src/P/Functor.hs b/src/P/Functor.hs
index d9dec1f..1923d65 100644
--- a/src/P/Functor.hs
+++ b/src/P/Functor.hs
@@ -33,33 +33,6 @@ import P.Function.Flip
 infixl 4 <$
 
 
-instance
-  (
-  )
-    => Functor ((,,,,) a b c d)
-  where
-    fmap f (x1, x2, x3, x4, x5) =
-      (x1, x2, x3, x4, f x5)
-
-
-instance
-  (
-  )
-    => Functor ((,,,,,) a b c d e)
-  where
-    fmap f (x1, x2, x3, x4, x5, x6) =
-      (x1, x2, x3, x4, x5, f x6)
-
-
-instance
-  (
-  )
-    => Functor ((,,,,,,) a b c d e f)
-  where
-    fmap f (x1, x2, x3, x4, x5, x6, x7) =
-      (x1, x2, x3, x4, x5, x6, f x7)
-
-
 {-# Deprecated fmap "Use m or (<) instead" #-}
 -- | Map a function over an arbitrary functor.
 -- Long version of 'P.Function.Compose.m' and '(P.Function.Compose.<)'
